stream_alert/rule_processor/rules_engine.py

```diff
@@ -274,12 +277,10 @@ def validate_datatypes(cls, normalized_types, datatypes):
             (boolean): return true if all datatypes are defined
         """
         if not normalized_types:
-            LOGGER.error('Normalized types dictionary is empty.')
             return False
 
         for datatype in datatypes:
             if not datatype in normalized_types:
-                LOGGER.error('The datatype [%s] is not defined!', datatype)
```

Comment:
Is this logging statement no longer helpful? I understand why the one above was removed but can you explain why this one was also?

Reply:
Since we removed `log_sources` constrain, one rule will matched to multiple records. Normalized types dictionaries are defined based on each log source though. It is not an error if normalized type `foo` doesn't defined in Record A, it just doesn't match and return `False` is enough. Does it make sense to you?